bookdown/05-technical.Rmd

```diff
@@ -96,35 +96,39 @@ learner = lrn("classif.debug")
 print(learner)
 ```
 
-This learner comes with special hyperparameters that let us control
+This learner comes with special hyperparameters that let us control:
 
-1. what conditions should be signaled (message, warning, error, segfault) with what probability,
-2. during which stage the conditions should be signaled (train or predict), and
```

Comment:
I think this was a deliberate choice so that it reads as a complete sentence although it has the enumeration. If you think it sounds too weird then you should also correct the punctuation, i.e. remove the point at item 3? Although i am never so sure about this punctuation but it is inconsistent now.